dev-util/cppunit/cppunit-1.14.0.recipe

```diff
@@ -1,7 +1,6 @@
 SUMMARY="C++ unit testing framework"
-DESCRIPTION="
-CppUnit is a C++ unit testing framework. It started its life as a port of \
-JUnit to C++ by Michael Feathers."
+DESCRIPTION="CppUnit is a C++ unit testing framework. It started its life as \
+a port of JUnit to C++ by Michael Feathers."
 HOMEPAGE="http://cppunit.sourceforge.net/"
```

Comment:
The linked `HOMEPAGE` only contains a link to some "FrontPage" which links to an empty page.

```diff
-HOMEPAGE="http://cppunit.sourceforge.net/"
+HOMEPAGE="https://www.freedesktop.org/wiki/Software/cppunit/"
```